wp-content/themes/core/parts/footer.html

```diff
@@ -1,5 +1,5 @@
-<!-- wp:group {"metadata":{"name":"Site Footer Pattern"},"align":"full","className":"site-footer alignfull is-style-dark","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"0","margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
-<div class="wp-block-group site-footer__container alignfull is-style-dark" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:group {"metadata":{"categories":["navigation","menu"],"patternName":"patterns/enhanced-nav-menu","name":"Enhanced Navigation Menu"},"align":"wide","className":"nav-menu__enhanced alignwide","style":{"spacing":{"blockGap":"var:preset|spacing|40","padding":{"right":"0","left":"0"}}},"layout":{"type":"default"}} -->
+<!-- wp:group {"tagName":"footer","metadata":{"name":"Site Footer Pattern"},"align":"full","className":"site-footer__container alignfull is-style-dark","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"0","margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<footer class="wp-block-group site-footer__container alignfull is-style-dark" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:group {"metadata":{"categories":["navigation","menu"],"patternName":"patterns/enhanced-nav-menu","name":"Enhanced Navigation Menu"},"align":"wide","className":"nav-menu__enhanced alignwide","style":{"spacing":{"blockGap":"var:preset|spacing|40","padding":{"right":"0","left":"0"}}},"layout":{"type":"default"}} -->
```

Comment:
❓ Did we mean to switch this back to a `footer` element? I thought this element already gets wrapped in a `footer` tag and that's why this one wasn't a footer tag. Could you double check this?